Load cldr data for "en" locale by default for Globalize + refactor

GoodDayForSurf opened the pull request:
<!--
Make sure that you've read the "Contributing Code and Content" section in CONTRIBUTING.md

If you are submitting a bug fix, the subject should contain "fixes ISSUE_ID" or "resolves ISSUE_ID".

In addition, please clarify:

1. What did you do? 
2. How did you do it?
3. How should we verify this?

-->

GoodDayForSurf committed:
Load cldr data for "en" locale by default for Globalize
+ remove inlined data
+ Use for Globalize cldr-data only from from package devextreme-cldr-data

nightskylark commented on testing/tests/DevExpress.localization/localization.globalize.widgets.tests.js:
```diff
@@ -230,14 +229,16 @@ QUnit.module('DateBox', commonEnvironment, () => {
-            assert.equal(cellText, '٢٦', 'Cell localized');
+
+            // NOTE: first cell text must be equal 31 because for AR locale first date of week is Saturday
+            // assert.equal(cellText, '٣١', 'Cell localized');
```
Why this code is commented on?

nightskylark commented on build/gulp/localization.js:
```diff
@@ -15,7 +16,11 @@ const Cldr = require('cldrjs');
-const parentLocales = require('../../node_modules/cldr-core/supplemental/parentLocales.json').supplemental.parentLocales.parentLocale;
+const parentLocales = require('cldr-core/supplemental/parentLocales.json').supplemental.parentLocales.parentLocale;
+
+/* for Globalize*/
```
Could you try to avoid such comments in the code and make the code self-documenting?

GoodDayForSurf committed:
Load cldr data for "en" locale by default for Globalize
+ remove inlined data
+ Use for Globalize cldr-data only from from package devextreme-cldr-data
